ogr/services/github.py

```diff
@@ -595,6 +596,25 @@ def get_releases(self) -> List[Release]:
             for release in releases
         ]
 
+    def create_release(
+        self,
+        tag: str,
+        name: str,
+        message: str,
+        draft=False,
+        prerelease=False,
+        target_commitish=GithubObject.NotSet,
```

Comment:
I would go with the subset of properties that can be implemented also in the other implementation.

(It would be nice to avoid Github specific objects in the signature of the method...)